recipes/metpy/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```

Comment:
Done. Those were in my original recipe, but I couldn't remember the purpose, so I pulled them while "cleaning up". **sigh**